src/internet_identity/tests/integration/v2_api/identity_register.rs

```diff
@@ -48,36 +47,29 @@ fn should_not_exceed_configured_identity_range() {
     create_identity_with_authn_method(&env, canister_id, &authn_method);
     create_identity_with_authn_method(&env, canister_id, &authn_method);
 
-    match_value!(
-        api_v2::captcha_create(&env, canister_id).unwrap(),
-        Some(CaptchaCreateResponse::Ok(challenge))
-    );
+    let challenge = api_v2::captcha_create(&env, canister_id).unwrap().unwrap();
 
-    match_value!(
-        api_v2::identity_register(
-            &env,
-            canister_id,
-            authn_method.principal(),
-            &authn_method,
-            &ChallengeAttempt {
-                chars: "a".to_string(),
-                key: challenge.challenge_key,
-            },
-            None,
-        ),
-        Ok(Some(IdentityRegisterResponse::CanisterFull))
-    );
+    let result = api_v2::identity_register(
+        &env,
+        canister_id,
+        authn_method.principal(),
+        &authn_method,
+        &ChallengeAttempt {
+            chars: "a".to_string(),
+            key: challenge.challenge_key,
+        },
+        None,
+    )
+    .unwrap();
```

Comment:
nit: there is a bunch of new `unwrap()`s in the tests. I think it is generally better to use `expect()`s and provide some context what has failed, as otherwise one has no idea from the log, especially if we have a chain `unwrap().unwrap()`.  Maybe add some context in some cases, where it would make sense?

Reply:
I replaced the `unwrap` calls now.